src/lib/csv/pacbio.js

```diff
@@ -0,0 +1,78 @@
+import { parse } from 'csv-parse/lib/sync'
+
+const castFloat = (value) => Number(value)
+const castString = (value) => String(value)
+
+// Handles the casting of columns. By default, unless specified here, a column
+// will be cast to a String
+const columnCasting = {
```

Comment:
It could do, but it's pretty closely tied to the underlying data models that I'm not sure there's much to gain. (Ie. A change in configuration would always be accompanied by changes in the code-base itself)

Maybe the casting is happening in the wrong place, and we should be performing it in the mutations? I had originally planned to cast the source column here as well, but ultimately decided that made more sense elsewhere (as it made error handling substantially easier) so maybe the same is true here.